src/common/darktable.c

```diff
@@ -1207,6 +1274,29 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
   return 0;
 }
 
+void dt_get_sysresource_level()
+{
+  darktable.dtresources.level = 0;
+  const char *config = dt_conf_get_string_const("resourcelevel");
+  /** These levels must correspond with preferences in xml.in **and** fractions
+      Please note: absolute values for "reference (debug)"
+  */
+  if(config)
+  {
+         if(!strcmp(config, "default"))           darktable.dtresources.level = 0; 
+    else if(!strcmp(config, "mini (debug)"))      darktable.dtresources.level = 1;
+    else if(!strcmp(config, "small"))             darktable.dtresources.level = 2;
+    else if(!strcmp(config, "medium"))            darktable.dtresources.level = 3;
```

Comment:
no more medium in darktableconfig.xml.in

Reply:
oh man, how did that happen on my side ...

Reply:
working on too many branches and forgot to push